CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# How to contribute
+
+First of all, thanks for contributing!
+
+This document provides some basic guidelines for contributing to this repository. To propose
+improvements or fix a bug, feel free to submit a PR.
+
+## Legal requirements
+
+Before we can accept your contributions you have to sign the [Contributor License Agreement][0]
```

Comment:
are we going to use cla-assistant for it, isn't it?

Reply:
yup, already in place, included Github integration